### DISCUSSION TITLE
Arcade PCB emulation for VMU/Mic

### DISCUSSION THREAD

**Discussion by danieldoyle** [OrangeFox86/DreamPicoPort#117] [General]
Would your work on this library be something that could be adapted to replicate the functionality of this board with a Pi Pico?
https://www.ebay.com/itm/166444459480

Sega part 839-1132 is basically a maple bus with a VMU connector that allowed the King of Route 66 arcade cabinet to use the DC microphone for radio communication.  Additionally it was used in Ferrari F355 for letting users bring their own saves on their VMU.  These are pretty rare parts (I have one however) and as you can see the price reflects that.  I was thinking if your work could be leveraged maybe something cheaper based on a pico could be produced.  Sourcing those DC VMU connectors may not be easy but maybe it could come from junk controllers (haven't researched to see if the part can be bought). Thoughts?

**Comment by Tails86** [comment]
If I'm understanding this module correctly (and it's very possible I'm not), it is acting as a main peripheral while allowing the sub peripheral to directly communicate when connected. It would take a little bit of tweaking if the current main peripheral client class was used since the assumption in that class is that all sub-peripherals are virtually instantiated. In this case, it's a physical add-on. It seems feasible though.

**Comment by danieldoyle** [comment]
Yup, you’ve got it.  NAOMI hardware has its own maple bus and this little
board was a way to take the DC controller peripherals I mentioned and feed
them into that.  So it was like a DC controller without any of the
controller parts
